### PR TITLE
forgot_password_view

### LINKED ISSUE #6: Forgot Password view and otp view
<h1>NOTE</h1>
create seperate branch named->"USERNAME" -"PAGE OR COMPONENT YOU ARE WORKING ON"

### ASIGNEE: @archeeetah

<h1>PREVIEW</h1>

image

### PR DESCRIPTION
**Pull request by archeeetah**
Fix #6 

https://github.com/user-attachments/assets/c8ea702a-bd67-4f27-8098-5c950bfb4935

### CODE REVIEW

File: lib/app/modules/password_view/controllers/password_view_controller.dart
Comment (SHREYANSHSK) [inline lib/app/modules/password_view/controllers/password_view_controller.dart:1]: add dispose methods as well to dispose controllers , this prevents memory leaks

**Review by SHREYANSHSK — CHANGES_REQUESTED** [review]
small change then i will merge , nicely work done, by far the best pr i have received from a member.

Good Job👍👍

File: lib/app/modules/password_view/controllers/password_view_controller.dart
Comment (archeeetah) [reply lib/app/modules/password_view/controllers/password_view_controller.dart:1]: Already added there bhaiya

Comment (archeeetah) [reply lib/app/modules/password_view/controllers/password_view_controller.dart:1]: can't add dispose method with getx controller ...should I do it by creating a stateful widget?

Comment (SHREYANSHSK) [reply lib/app/modules/password_view/controllers/password_view_controller.dart:1]: ok no problem i am merging the pr

**Pull request merged**